string id
string idRoom
string type
float64 score
geometry_msgs/Pose pose
geometry_msgs/Vector3 scale
sensor_msgs/PointCloud pointCloud
